src/components/buildUrl.brs

```diff
@@ -0,0 +1,32 @@
+' Compose url with query strings. Values are encoded.
+' @example
+' ?buildUrl("http://myurl.com", { queryString: "123" })
+' ' prints http://myurl.com?queryString=123
+' @param {String} path
+' @param {Object} [params=Invalid] - Query strings. The AA of strings.
+' @returns {String}
+function buildUrl(path as String, params = Invalid as Object) as String
+  if (params = Invalid OR params.count() = 0)
+    return path
+  end if
+
+  paramParts = []
+  ' It might be encoded already. To avoid encoding encoded url it needs to be decoded first.
+  encodedPath = path.decodeUri().encodeUri()
```

Comment:
I changed the implementation a bit. I noticed it was not safe to use because it did not encode `path` properly. Let me know what you think.